### PR TITLE
feat: quiz listing page

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**

### CODE REVIEW

**Review by Rishi05051997 — APPROVED** [review]
Please remove spaces in HTML, other things are ok for me

File: pages/quizzes.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            <label for="NodeJS">Node JS</label>
+          </div>
+          <div>
+            <input type="checkbox" id="HTML" name="category" value="HTML" checked>
+            <label for="HTML">HTML</label>
+          </div>
+          <div>
+            <input type="checkbox" id="CSS" name="category" value="CSS">
+            <label for="CSS">CSS</label>
+          </div>
```
Comment (amirnathani9) [inline pages/quizzes.html:90]: why the id in capital letters?

**Review by amirnathani9 — APPROVED** [review]
Your code is clean and looks good.
Add a description with a preview link.
Your deployed netlify preview link is not working

File: pages/quizzes.html (hunk 1 above)
Comment (HarshnaNHaswani) [reply pages/quizzes.html:90]: Thanks for pointing it out, I'll change it

**Comment by HarshnaNHaswani** [comment]
Thank you @amirnathani9  @Rishi05051997 for the PR, will make the necessary changes

**Pull request merged**